requirements-dev.txt

```diff
@@ -3,7 +3,7 @@ cmake!=3.17.1,!=3.17.0
 conan<2.0.0
 scikit-build>=0.11.0
 asv
-cvxpy<1.5.0
+cvxpy<1.6.0
```

Comment:
cvxpy 1.5.4 also doesn't have Python 3.13 wheels (though it's just a test dependency, not a build one).  If 1.5.4 works, is there any chance that 1.6.0 works too, and we can just completely remove the upper bound?

Reply:
I'm reluctant to remove upper bounds in a project like Aer which is in maintenance mode, so the main pitfalls to handle are incompatible new version of libraries.

Reply:
Yeah, that seems like a good idea for the dev requirement.